pkglistgen/tool.py

```diff
@@ -757,28 +759,32 @@ def update_and_solve_target(
                 checkout_package(api.apiurl, project, package, expand_link=True,
                                  prj_dir=cache_dir, outdir=os.path.join(cache_dir, package))
 
-        if not os.path.isdir(product_dir):
+        if use_product_composer:
```

Comment:
Since we now check against "product-composer" mode every time rather than looking at directories, I think we can drop this if block.

Reply:
yes, indeed, fine for me